platform/runner-extension/std/sys/src/main/kotlin/io/hamal/extension/std/sys/Extension.kt

```diff
@@ -34,6 +36,8 @@ class SysExtensionFactory(
                 "await_failed" to AwaitFailedFunction(httpTemplate),
 
                 "adhoc" to AdhocInvokeFunction(sdk),
+                "code_get" to CodeGetFunctionI(sdk),
+                "code_get_version" to CodeGetFunctionII(sdk),
```

Comment:
Rule of Thumb - if you ever finding yourself in a place where you need to have FunctionOne and FunctionTwo - you usually doing something wrong 

```suggestion
                "code_get" to CodeGetFunction(sdk)
```